src/auro-menu.js

```diff
@@ -32,7 +34,8 @@ class AuroMenu extends LitElement {
 
   static get properties() {
     return {
-      optionSelected: { type: Object }
+      optionSelected: { type: Object },
+      matchWord: { type: String }
```

Comment:
I am not seeing this being used in the DOM, is this a @private API attribute?

Reply:
No, it's not in the DOM. Please see the working demo showing how it is used.

Reply:
That's my question... I see how it's used. Since it is not active in the DOM, that would mean it's a @private attribute? So it shouldn't appear in the API docs?